cg/meta/workflow/microsalt/quality_controller/utils.py

```diff
@@ -0,0 +1,142 @@
+from pathlib import Path
+
+from cg.constants.constants import MicrosaltAppTags, MicrosaltQC
+from cg.meta.workflow.microsalt.constants import QUALITY_REPORT_FILE_NAME
+from cg.meta.workflow.microsalt.metrics_parser.models import SampleMetrics
+from cg.meta.workflow.microsalt.quality_controller.models import (
+    CaseQualityResult,
+    SampleQualityResult,
+)
+from cg.meta.workflow.microsalt.quality_controller.result_logger import ResultLogger
+from cg.models.orders.sample_base import ControlEnum
+from cg.store.models import Sample
+
+
+def is_valid_total_reads(reads: int, target_reads: int) -> bool:
+    return reads > target_reads * MicrosaltQC.TARGET_READS_FAIL_THRESHOLD
```

Comment:
This is incorrect, it should be 75% of the application tag reads. The documentation table in [atlas](https://atlas.scilifelab.se/production/data_analysis/workflows/microsalt/#quality-control-criteria) follows what microsalt gives as output, however, below it we have an anecdote regarding the reads and that we have the requirement of 75% per sample.